documentation.md

```diff
@@ -0,0 +1,233 @@
+# Cyberwatch API Toolbox documentation
+
+## Available methods
+
+#### Ping
+
+Send a GET request to `/api/v2/ping` to validate that the API is working.
+
+###### Usage example and expected result:
+
+```python
+>>> CBWApi(URL, API_KEY, SECRET_KEY).ping()
+INFO:root:OK
+True
+```
+
+#### Servers
+
+Send a GET request to `/api/v2/servers` to retrieve the list of all servers.
+
+###### Usage example and expected result:
+
+```python
+>>> CBWApi(URL, API_KEY, SECRET_KEY).servers()
+[<cbw_api_toolbox.cbw_objects.cbw_server.CBWServer ...]
+```
+
+#### Server
+
+Send a GET request to `/api/v2/servers/{SERVER_ID}` to retrieve the information of a server.
+
+##### Usage example and expected result:
+
+```python
+>>> CBWApi(URL, API_KEY, SECRET_KEY).server(SERVER_ID)
+[<cbw_api_toolbox.cbw_objects.cbw_server.CBWServer]
+```
+
+#### Update server
+
+Send a PATCH request  `/api/v2/servers/{SERVER_ID}` to update the information of a server.
+
+```python
+>>> CBWApi(URL, API_KEY, SECRET_KEY).update_server(SERVER_ID, INFO)
+True
+```
+
+#### Delete server
+
+Send a DELETE request `/api/v2/servers/{SERVER_ID}` to delete SERVER_ID.
+
+```python
+>>> CBWApi(URL, API_KEY, SECRET_KEY).delete_server(SERVER_ID)
+True
+```
+
+#### Remote accesses
+
+Send a GET request `/api/v2/remote_accesses` to retrieve the list of all remote accesses.
+
+```python
+>>> CBWApi(URL, API_KEY, SECRET_KEY).remote_accesses()
+[<cbw_api_toolbox.cbw_objects.cbw_remote_access.CBWRemoteAccess...]
+```
+
+#### Remote access
+
+Send a GET request `/api/v2/remote_accesses/{REMOTE_ACCESS_ID}` to retrieve the information of a particular remote access.
+
+```python
+>>> CBWApi(URL, API_KEY, SECRET_KEY).remote_access(REMOTE_ACCESS_ID)
+[<cbw_api_toolbox.cbw_objects.cbw_remote_access.CBWRemoteAccess]
+```
+
+#### Create remote access
+
+Send a POST request `/api/v2/remote_accesses` to create a remote access
```

Comment:
Sometimes you put a dot " . " as the end of the sentence, sometimes you don't
- put the dot on every sentence that requires one
- make sure this is applied to the entire file (i'm not commenting on every missing dot)